components/tor/tor_control.cc

```diff
@@ -397,7 +411,7 @@ void TorControl::GetVersionLine(std::string* version,
     VLOG(0) << "tor: unexpected " << kGetVersionCmd << " reply";
     return;
   }
-  *version = reply.substr(strlen(kGetVersionReply));
+  *version = reply.substr(length(kGetVersionReply));
```

Comment:
`strlen` should work as intrinsic in many cases, so the old code works pretty much the same:
https://godbolt.org/z/dW91jhcbz

Reply:
agree with @goodov, I don't see the need to replace strlen with length